packages/bpk-component-scrollable-calendar/readme.md

```diff
@@ -8,6 +8,34 @@
 npm install bpk-component-scrollable-calendar --save-dev
```

Comment:
As this components' usage will look similar to the underlying component, it may make more sense to put a link to https://backpack.github.io/components/calendar/?platform=web#readme rather than duplicating a lot of it.

Otherwise if we change the underlying component they might get out of sync

The same probably applies for the proptypes that just get passed down.
You can see an example of where we've done this when we've wrapped components ourselves:
https://backpack.github.io/components/autosuggest/?platform=web#props